core/lib/dal/migrations/20231018205126_l1_batches_boojum_fields.up.sql

```diff
@@ -0,0 +1,6 @@
+ALTER TABLE l1_batches
+    ADD COLUMN IF NOT EXISTS system_logs BYTEA[]
+    NOT NULL DEFAULT '{}';
+
+ALTER TABLE l1_batches
+    ADD COLUMN IF NOT EXISTS compressed_state_diffs BYTEA;
```

Comment:
@RomanBrodetski A question I've wanted to ask for some time: Is it not feasible to have a catch-all `jsonb` column in `l1_batches` (and maybe other tables) so that we don't need to perform migrations each time we add a new piece of data for the corresponding entity?

Reply:
makes sense esp for fields not used relationally